README.md

```diff
@@ -321,17 +372,28 @@ For this reason, we also decided to implement a thread safe implementation of th
 
 ### Impact on the data model
 
-To apply transaction, we can just add a keyword `TRANSACTION` before table name in Scalar DB scheme. For instance, we modify our qa.question talbe.
+To apply transaction, we can just add a key `transaction` and value as `true` in the Scalar DB scheme. For instance, we modify our qa.question schema.
```

Comment:
`database/schema_storage.json` has a member `transaction`.
I think it is better to modify this context or remove `transaction` from the json file.
(`transaction` is `false` by default if you don't specify `transaction` in the schema file.)

Reply:
Thank you for the suggestion.
Document update accordingly.